java/arcs/sdk/android/storage/service/StorageService.kt

```diff
@@ -147,7 +154,7 @@ open class StorageService : ResurrectorService() {
         }
 
         if (intent.action == DEVTOOLS_ACTION) {
-            return DevToolsStorageManager(coroutineContext, stores)
+            return DevToolsStorageManager(coroutineContext, stores, devToolsProxy)
         }
```

Comment:
Maybe this conditional should also have a check for `ApplicationInfo.FLAG_DEBUGGABLE != 0`?